domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Controllers/ConnectionInvitationsControllerTests.cs

```diff
@@ -46,4 +42,77 @@ public async Task CreateConnectionInvitation_ShouldReturnUnauthorized_WhenUnauth
 
         result.StatusCode.Should().Be(HttpStatusCode.Unauthorized);
     }
+
+    [Fact]
+    public async Task GetConnectionInvitation_ShouldReturnOK_WhenInvitationExists()
+    {
+        var invitationId = Guid.NewGuid();
+        var invitation = new ConnectionInvitation
+        {
+            Id = invitationId,
+            SenderCompanyId = Guid.NewGuid(),
+            SenderCompanyTin = "12345678"
+        };
+
+        await factory.SeedConnectionInvitations(new List<ConnectionInvitation> { invitation });
```

Comment:
No need to seed - you can call POST first then GET after

Reply:
fixed in https://github.com/Energinet-DataHub/energy-origin/pull/415/commits/f040ef535680294e0a47ae5624b620361a4cebcf